tidy3d/components/geometry.py

```diff
@@ -19,8 +19,17 @@
 from .viz import add_ax_if_none, equal_aspect
 from .viz import PLOT_BUFFER, ARROW_LENGTH, arrow_style
 from .viz import PlotParams, plot_params_geometry, polygon_patch
-from ..log import Tidy3dKeyError, SetupError, ValidationError, log
+from ..log import Tidy3dKeyError, SetupError, ValidationError, log, DataError
 from ..constants import MICROMETER, LARGE_NUMBER, RADIAN, fp_eps, inf
+from .data.dataset import SurfaceMeshDataset
+from .data.data_array import SurfaceMeshDataArray, DATA_ARRAY_MAP
+
+try:
+    import trimesh
+
+    TRIMESH_AVAILABLE = True
+except ImportError:
+    TRIMESH_AVAILABLE = False
```

Comment:
Consider one situation: the user has trimesh installed, but not networkx and rtree. So it will not error here, but will have issues in plotting. Should we add another `try ...` that raises an error in `intersections_plane`? @tylerflex @caseyflex

Reply:
I'm not clear on where `rtree` and `networkx` are used. Are they something that `trimesh` uses internally when calling certain methods? We could try importing them in this try/ except block, or maybe `try` when we call the `trimesh` method that uses them and except `ImportError` there?